### ISSUE TITLE
Can't read openvino pre-trained model in opencv.

### ISSUE THREAD

**Issue by chinmaydas96** [opencv/open_model_zoo#1022]
I was running the following line :
`
import cv2                                                                                                                                           

In [2]: cv2.dnn.readNet('person-detection-retail-0002/person-detection-retail-0002.xml',  
   ...:    ...:                             'person-detection-retail-0002/person-detection-retail-0002.xml')   
`

the error is following : 

** 
terminate called after throwing an instance of 'InferenceEngine::details::InferenceEngineException'
  what():  Cannot create Const layer with name: 411/Output_0/Data__const. Layer has incorrect weights!
Aborted
**

I have already build the opencv with openvino.
`import cv2 as cv                                                                                                                                     

In [2]: print(cv.__file__)                                                                                                                                   
/home/pi/openvino/python/python3/cv2.abi3.so`

**Comment by maxl16** [comment]
Hello @chinmaydas96 

There's a known case with the latest models due to changes in the Model Optimizer and ngraph in latest OpenVINO toolkit 2020.1 build. The "axis" error is caused by the new IR v10 format. 

As a workaround, you can use one of the following:
1) Either use the models from the previous release (2019 R3) of open model zoo  - https://download.01.org/opencv/2019/open_model_zoo/R3/20190905_163000_models_bin/
2) Or generate the IRv7 format models using the Model Optimizer from the OpenVINO toolkit 2020.1 build with the following parameter: --generate_deprecated_IR_V7

Please let us know if this works for you or not.

Thanks.
Best regards, Max.

**Comment by IRDonch** [comment]
This looks like an issue with OpenCV, so I'm moving it to the OpenCV repo.
